ironfish-cli/src/commands/wallet/send.ts

```diff
@@ -32,7 +32,7 @@ export class Send extends IronfishCommand {
     ...RemoteFlags,
     account: Flags.string({
       char: 'f',
-      description: 'The account to send money from',
+      description: 'The account to send coins from',
```

Comment:
coins is more consistent with the description of the command and of other fields in the command